Fixed an issue where the shopping cart remained empty due to a header…

younes-dro opened the pull request:
… modification error.

Add file : languages\ets_q_n_a.pot  pulled with 'SVN UP' command

younes-dro committed:
Fixed an issue where the shopping cart remained empty due to a header modification error.

Add file : languages\ets_q_n_a.pot  pulled with 'SVN UP' command

younes-dro committed:
Update plugin version and changelog : version  1.2.7

geek92sunny commented on includes/ets_user_qa_function.php:
```diff
@@ -1,193 +1,194 @@
+<?php
```
The blank space on the start of the file was the only change we did here?

younes-dro replied:
Yes

geek92sunny commented on languages/ets_q_n_a.pot:
```diff
@@ -0,0 +1,163 @@
+# Copyright (C) 2023 ExpressTech Software Solutions Pvt. Ltd.
```
Discard this POT, this file should have same name as the text domain.

geek92sunny commented on includes/ets_user_qa_function.php:
```diff
@@ -196,380 +197,411 @@ public function ets_ask_qustion_tab() {
 						<tr>
-							<td class="ets-question-title"><p><?php echo __('Answer','ets_q_n_a'); ?>:</p></td>
-							<td colspan="2"><p> <?php echo $value['answer'];?></p></td> 
+							<td class="ets-question-title"><p><?php echo __( 'Answer', 'ets_q_n_a' ); ?>:</p></td>
```
Please update text domain in translation func here.

younes-dro replied:
> Please update text domain in translation func here.

Already updated.
dont find any text domain : 'ets_q_n_a' in the plugin files

geek92sunny commented on includes/ets_user_qa_function.php:
```diff
@@ -196,380 +197,411 @@ public function ets_ask_qustion_tab() {
 						<tr>
-							<td class="ets-question-title"><p><?php echo __('Answer','ets_q_n_a'); ?>:</p></td>
-							<td colspan="2" class="ets-no-answer"><h6><p><i><?php echo __("Answer awaiting",'ets_q_n_a');?>...</i></p></h6></td>	
+							<td class="ets-question-title"><p><?php echo __( 'Answer', 'ets_q_n_a' ); ?>:</p></td>
```
Please update text domain name here too.

geek92sunny commented on readme.txt:
```diff
@@ -85,4 +85,7 @@ No configuration needed! just install and it will start working from there, you
\ No newline at end of file
+- Dev: Added the shortcode [display_qa_list]
+
+= 1.2.7 =
```
Rephrase changelog  "Fixed the blockage in the AJAX requests."

geek92sunny replied:
Please rephrase the 1.2.7 as mentioned above.

geek92sunny reviewed (changes requested):
A question and some change requests, version these as 1.2.8.

younes-dro committed:
Update Text domain with the correct slug @geek92sunny

geek92sunny reviewed (changes requested):
We already have a `product-questions-answers-for-woocommerce.pot` file under language directory, please update that and discard the new POT file you created.

geek92sunny commented:
Please do the pending changes and update the version to 1.2.8 @younes-dro

younes-dro committed:
rephrase the 1.2.7 changelog

younes-dro committed:
Remove unnecessary file: languages/ets_q_n_a.pot

geek92sunny commented:
Since we also changed the text domain in some places, can you please change the version to 1.2.8 ?

The changelog will be 
= 1.2.6 =
\- Dev: Replaced old text domain to some places

younes-dro commented:
> Since we also changed the text domain in some places, can you please change the version to 1.2.8 ?
> 
> The changelog will be = 1.2.6 = - Dev: Replaced old text domain to some places

@geek92sunny you mean 1.2.8 NOT 1.2.6 ?

younes-dro committed:
Update plugin version to 1.2.8

geek92sunny reviewed (approved):
All set!
